src/components/App/App.css

```diff
@@ -2,6 +2,18 @@
   text-align: center;
 }
 
+.tree-profile{
+  padding-bottom: 30px; /* Leaves space between profile and tree */
```

Comment:
Can remove comment here

Reply:
Fixed both in the recent push before merging :D